contracts/upgradeable_contracts/ERC677Bridge.sol

```diff
@@ -1,15 +1,25 @@
 pragma solidity 0.4.24;
 
 import "./BaseERC677Bridge.sol";
+import "../libraries/Bytes.sol";
 
 contract ERC677Bridge is BaseERC677Bridge {
+    function chooseReceiver(address _from, bytes _data) internal view returns (address recipient) {
```

Comment:
Seems that the same code appeared in few modules. Raised similar comment [here](https://github.com/poanetwork/poa-bridge-contracts/pull/302#discussion_r337170977)

Reply:
Correct, after https://github.com/poanetwork/poa-bridge-contracts/pull/302 is merged, this PR will be updated to avoid repetition of the same methods used